src/translations/en.json

```diff
@@ -770,6 +772,11 @@
   "SavedCardDetails.replaceCardTitle": "Use another payment card",
   "SavedCardDetails.savedPaymentMethodPlaceholderDesktop": "•••• •••• •••• {last4Digits}",
   "SavedCardDetails.savedPaymentMethodPlaceholderMobile": "…{last4Digits}",
+  "SearchCTA.buttonLabel": "Search",
+  "SearchCTA.dateFilterPlaceholder": "Dates",
+  "SearchCTA.keywordFilterPlaceholder": "Search listings",
+  "SearchCTA.locationPlaceholder": "Location",
+  "SearchCTA.currentLocationPlaceholder": "Current location",
```

Comment:
Perhaps these should also be namespaced 
E.g. "Pages.SearchCTA.*" or "PageBuilder.SearchCTA.*"